core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -394,15 +399,13 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.expectNumberOfSkillsToBe = async function(number) {
+    await this.filterSkillsByStatus('Unassigned');
```

Comment:
Is it possible to avoid mutating the page state in this function? If not, can we either:
1. restore the state at the end of the function
2. change the function name to reflect the change it performs

Reply:
Separated these also.